cranelift/filetests/filetests/vcode/aarch64/call.clif

```diff
@@ -16,3 +16,32 @@ block0(v0: i64):
 ; nextln:  mov sp, fp
 ; nextln:  ldp fp, lr, [sp], #16
 ; nextln:  ret
+
```

Comment:
I'm not 100% sure what I'm looking at here, but I was a bit mystified in that I can't see any signed widening insns in the expected outputs.  But it looks like the patch above handles both signed and unsigned widening.  If a signed-widening version of this test case does make sense, please can you add it?

Reply:
+1; this only tests uext, not sext, which would be nice to have tests for!

Reply:
Added `sext` tests as well; thanks!